datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -396,15 +412,20 @@ def apply(self):
         self._pcd_dir = osp.join(self._save_dir, self.PATH_CLS.PCD_DIR)
         self._related_images_dir = osp.join(self._save_dir, self.PATH_CLS.RELATED_IMAGES_DIR)
 
-        writers = {subset: self.create_writer(subset) for subset in self._extractor.subsets()}
+        writers = {
+            subset: self.create_writer(
+                subset, self._images_dir, self._pcd_dir, self._related_images_dir
+            )
+            for subset in self._extractor.subsets()
+        }
 
         for writer in writers.values():
             writer.add_infos(self._extractor.infos())
             writer.add_categories(self._extractor.categories())
 
         for item in self._extractor:
             subset = item.subset or DEFAULT_SUBSET_NAME
-            writers[subset].add_item(item)
+            writers[subset].add_item(item, pool)
```

Comment:
How about just accepting all arguments since `_SubsetWriter` could parents of other than one from `datumaro_binary`?
```suggestion
            writers[subset].add_item(item, *args, **kwargs)
```

Reply:
Same here.

Reply:
Same here.